config/config.go

```diff
@@ -46,7 +46,7 @@ var (
 	defaultConfigFileName  = "config.toml"
 	defaultGenesisJSONName = "genesis.json"
 
-	defaultMode             = ModeFull
+	defaultMode             = ModeValidator
```

Comment:
This defeats part of the purpose of preventing people from accidentally spinning up a validator node and double signing. Are we opposed to setting the node to validator, if the validator set is 1?

Reply:
Well double signing can only happen if they `unsafe-reset-all` or find some other way to start a second validator with the same private key. Also just because they are running a validator, doesn't mean that they are in the validator set. I'm not saying that I disagree with you that there are no risks. I'm just providing some counter arguments. We can leave this as a draft and perhaps discuss it on Monday's call.